### PR TITLE
chore: iputils-ping is added to the devcontainer (needed by python services)

### PR DESCRIPTION
**Pull request by vktng**
Signed-off-by: Krisztián Varga <krisztian.varga@tngtech.com>

## Summary

First step to add iputils-ping dependency to the devcontainer (needed for python services).

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by LKreutzer — APPROVED** [review]
Lgtm

**Review by nstng — APPROVED** [review]
in pairing

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -56,6 +56,8 @@ RUN echo "Install general purpose packages" && \
         g++ \
         g++-9 \
         gcc-9 \
+        # dependency of python services (e.g. magmad)
```
Comment (electronjoe) [inline .devcontainer/Dockerfile]: Nit / feedback: is there any way to make this comment more clearly pertain to just iputils-ping? As it stands now I might wonder if I were a novice looking at this list.  Is it possible to `iputils-ping \ # dependency of python services (e.g. magmad)`? If not, maybe we say on line 59 `# iputils-ping is a dependency of python services (e.g. magmad)`? What do you think.

Comment (vktng) [reply .devcontainer/Dockerfile]: I would have preferred the first solution, but the highlighting was not correct, so we opted for the second option.

**Review by Neudrino — APPROVED** [review]

**Review by themarwhal — APPROVED** [review]

**Pull request merged**